Infrastructure/PluginExecutor.cs

```diff
@@ -42,16 +49,24 @@ public void Run(string message, string user)
                     || CheckIfIlegalPluginPressed(message, out int pluginNumber, out msgForUser))
```

Comment:
You created a function so get the current plugin, instead of loading it explicitly like this
`_dal.LoadPluginData(user, SESSION_PLUGIN_ID)`
You can call the function you created

This is important because in this way, you only need to change logic in one place if need to...

Reply:
OK